pyomo/core/base/sets.py

```diff
@@ -27,7 +27,7 @@
 from pyomo.common.deprecation import deprecation_warning
```

Comment:
It looks like `black` might have gone a bit crazy on this file. Can you please revert it?